File: web3rpc/rpc-specs/code-samples/javascript/klay/block/getStorageAt.js
const OpenSdk = require("opensdk-javascript");

(() => {
    const sdk = new OpenSdk(new OpenSdk.ApiClient("https://api.baobab.klaytn.net:8651"));

    const storageAddress = '0x295a70b2de5e3953354a6a8344e616ed314d7251'
    const position = '0x0'
    const blockNumberOrHash = 'latest'
    
    sdk.klay.getStorageAt(storageAddress, position, blockNumberOrHash, {}, (err, data, response) => {
        console.log(data);
    });


}
)()

File: web3rpc/rpc-specs/code-samples/python/klay/block/getStorageAt.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

address = "0x295a70b2de5e3953354a6a8344e616ed314d7251"
position = "0x0"
blockHash = "latest"

w3 = Web3(Web3.HTTPProvider(host))
klay_response = w3.klay.get_storage_at(address, position, blockHash)

print(klay_response)
